target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10mp.dts

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+#include "rtl83xx_d-link_dgs-1210_poe.dtsi"
+
```

Comment:
Shouldn't this also include the gpio related dtsi?

Reply:
It does include the gpio dtsi. This is described in the commit message at note 6. Or just as easy in the top of the poe.dtsi, poe can't exist without the hardware in gpio.dtsi.

Reply:
Also see the commit message part: 'How to implement in dts file settings:'

Reply:
I think @RaylynnKnight is trying to say 'include what you use'.

It's bad practice to rely on includes of other includes. What if the POE stuff doesn't include it anymore? Now your thing is broken as well.

Reply:
Has been changed.